scripts/miro_updates.py

```diff
@@ -264,6 +263,27 @@ def suppress_image(*, miro_id, message: str):
     _remove_image_from_cloudfront(miro_id=miro_id)
 
 
+def unsuppress_image(*, miro_id: str, origin: str, message: str):
+    """
+    Reinstate a hidden Miro image
+    """
+    sns_client = SESSION.client("sns")
+    topic_arn = _get_reindexer_topic_arn()
+    if not has_subscriptions(sns_client, topic_arn=topic_arn):
+        print(
+            "Nothing is listening to the reindexer, this action will not have the expected effect, aborting"
+        )
+        exit(1)
+
+    # First, make the DDS record reflect that the image should be visible
+    _set_image_availability(miro_id=miro_id, message=message, is_available=True)
+
+    # Now the actual image must be registered on DLCS so that it can be seen
+    register_on_dlcs(origin_url=origin, miro_id=miro_id)
+    # Finally, reindex the Miro record. This makes the Image record available to the API.
+    _request_reindex_for(miro_id=miro_id)
```

Comment:
I think this is done inside `_set_image_availability` also, so perhaps redundant?